feature-libs/my-account/organization/occ/converters/occ-org-unit-address-normalizer.ts

```diff
@@ -1,10 +1,10 @@
 import { Injectable } from '@angular/core';
-import { Converter, Occ, B2BAddress } from '@spartacus/core';
+import { Address, Converter, Occ } from '@spartacus/core';
 
 @Injectable()
 export class OccOrgUnitAddressNormalizer
```

Comment:
Same as with the normalizer above. Let's make it `OccAddressNormalizer` and move it to core.